dokka-integration-tests/settings.gradle.kts

```diff
@@ -36,7 +42,7 @@ dependencyResolutionManagement {
 includeBuild("../dokka-runners/runner-gradle-plugin-classic")
 includeBuild("../dokka-runners/runner-maven-plugin")
 includeBuild("../dokka-runners/runner-cli")
-includeBuild("../.") // include the very root aggregating build so that we can depend on its tasks
+includeBuild("../.")
```

Comment:
I removed this comment because it's no longer accurate: dokka-integration-tests depends on dependencies from the root project (the same as the other includedBuild dependencies).

Reply:
Does it make sense to update the comment, if this line is still needed, but for a different reason? I added it initially not so much to explain the task dependency, but the reason for depending on `../.` in general, as it's confusing as hell when you see it without context :)

Reply:
Sure thing! [I've started a suggestion](https://github.com/Kotlin/dokka/pull/3433/files#r1500933769) - edit or approve, whatever makes more sense to you.